scss/_lesson.scss

```diff
@@ -7,6 +7,24 @@
         color: $accent-color;
     }
     h2, h3 {
-        color: $brand-color;
+        color: $header-color;
     }
+}
+
+section ul {
+    list-style: none; /* Remove default bullets */
+    margin-right: 1em;
+}
+
+section ul li::before {
+    content: "\25CF";  /* \2022 is unicode for a bullet */
+    color: $brand-color; 
+    font-weight: bold; 
+    display: inline-block; /* Needed to add space between the bullet and the text */
+    width: 1em; /* Also needed for space */
+    margin-left: -1em; /* Also needed for space */
+}
+
+section ul li ul li::before {
```

Comment:
Is this supposed to be "section ul li ul li"?
If not, it can be combined with the group above this one

Reply:
I have tried with section ul: last-child li::before but id didn't work with couple first paragraphs, so I added full path. It is for sub-lists bullets.